maven-surefire-plugin/src/site/apt/examples/toolchains.apt.vm

```diff
@@ -54,3 +54,36 @@ Using Maven Toolchains with ${thisPlugin}.
 +---+
 
          The above example assumes that your toolchains.xml contains a valid entry with these values.
+
+* Minimal Java Versoin for Toolchains
+
+    Unfortunately, the minimum version of Java that can be used as a toolchain is limited by the Surefire Booter requirement.
+
+*---------------------------------------------+---------------------------+-------------------------+
+|| <<Surefire Version>>                       || <<Minimal Java Version>> || <<Class File Version>> |
+*---------------------------------------------+---------------------------+-------------------------+
+| 3.0.0-M6 or higher                          |    Java 8                 |    52                   |     
+*---------------------------------------------+---------------------------+-------------------------+
+| 3.0.0-M5 and lower                          |    Java 7                 |    51                   |  
+*---------------------------------------------+---------------------------+-------------------------+
+| To Be Completed                             |    Java 6                 |    50                   |    
+*---------------------------------------------+---------------------------+-------------------------+
+
+    If you try to use an older version than minimal supported one, you should face something like: 
```

Comment:
then the minimal

Reply:
You want me to change it like this : 
```diff
-If you try to use an older version than minimal supported one
+If you try to use an older version then the minimal supported one
```

My english is not very good but I didn't get the sentence like this, so I need some explanation ? 

I tried to get help with deepl and it propose something like : 

> If you try to use a version older than the minimum supported version

Does it sounds better  ?

Reply:
That is fine